install-aws-ofi-nccl.sh

```diff
@@ -1,5 +1,19 @@
 #!/bin/bash
 
+os_name="$(. /etc/os-release; echo $NAME)"
```

Comment:
So we want to split the dependency installation to each software installation scripts?